Documentation/Blazorise.Docs/Pages/Blog/2024-07-19_HowToStyleBlazoriseComponents/Index.md

```diff
@@ -0,0 +1,162 @@
+﻿---
+title: How to style Blazorise components
+description: Discover
+permalink: /blog/how-to-style-blazorise-components
+canonical: /blog/how-to-style-blazorise-components
+image-url: /img/blog/2024-07-19/how-to-style-blazorise-components.png
+image-title: How to style Blazorise components
+author-name: Giorgi
+author-image: giorgi
+posted-on: Jul 19th, 2024
+read-time: 7 min
+---
+
+# Styling Blazorise components
+
+Hello! today we are going to go over how to style Blazorise components!
+
+As your know, Blazorise is a framework-agnostic library, this means we have a lot of options when it comes to choosing what type of framework we want to use.
+
+Check out the [quick start](https://blazorise.com/docs/start) guide here to get started with Blazorise.
+
+## In this article, we will go over the following:
+
+- Styling Blazorise components#How does CSS work with Blazorise?|How does CSS work with Blazorise?
+- Styling Blazorise components#How to style Blazorise components?|How to style Blazorise components?
+- Styling Blazorise components#Limitations of CSS Isolation|Limitations of CSS Isolation
+
+---
+
+So let's dive into the topics and explore our options when it comes to styling Blazorise components.
+
+## How does CSS work with Blazorise?
+
+Blazorise, just like plain Blazor, is a framework that helps us generate responsive web UI. This means that Blazorise supports every CSS property that is supported by the browser. There are no special CSS properties that only apply to Blazor or Blazorise.
+
+If you would like to find out how to give your Blazorise application a different theme, check out [the docs about theming](https://blazorise.com/docs/theming)
+
+### Inline styling
+
+Inline CSS styling can be applied to any Blazor component directly.
```

Comment:
We proceed to talk about the basics of html / css... Let's imagine you're doing this for the sake of not assuming every user will know these. The point is they don't need to, they just want a component framework where they don't need to know this stuff. Also I would assume most Blazorise users would at least know this much.

As a Blazorise user I would be here for the intrinsicasies of Blazorise and how to use Blazorise and its components and how it can help me be productive.
How there are helper classes, fluent ui based apis, generic configuration, etc...

Reply:
But maybe it's just me. I'd also ask @stsrki what he thinks of this info.

Reply:
I agree. We always need to keep in mind that user not know everything. And guide them through the process. Whatever the process would be.

Reply:
@stsrki 

Your comment seems contradictory.

So you agree with @ddjerqq text instead? We should explain these html, css basics. basically I was advocating for us to focus more on Blazorise, then on the details (Personally If I was a developer using Blazorise I wouldn't find value in these first paragraphs, I want to learn Blazorise, not html, css). 

But again, I might be wrong and this might be useful information.

Reply:
No, I meant we need to guide users into Blazorise more. But, even if regular HTMS or CSS is explained it should still be fine. Not everyone knows it. Especially for someone that comes from backend background.

To summarize:

- explain Blazorise perspective more,
- explain regular HTML/CSS when needed

Reply:
Let's imagine I'm from backend, I want to see Blazorise information. If the blog is providing me this information that will probably not help me get value of Blazorise, I might leave the blog or loose interest.

There's already ton of html, css resources on the web if I needed.

My point is that although the information might be helpful, it's adding density to the text without providing "blazorise value".


Let's imagine that .NET as a product comes out, and instead of explaining how you can write safe code easily and how you can use efficiently, you have a blog talking about garbage collection, heap, etc... the purpose is that on the managed language you don't need to know about it, although it HELPS alot knowing how stuff works under the covers, that does not warrant putting that information there.

Reply:
> Let's imagine that .NET as a product comes out, and instead of explaining how you can write safe code easily and how you can use efficiently, you have a blog talking about garbage collection, heap, etc...

Hmmm. I get your point. I agree it could add noise if used too much. We can either just focus on Blazorise or, for reusability, write more blogs that explain some things in detail and then just provide a link.

But yeah. More focus on Blazorise as a product.